src/filtrering/filtrering-filter/filterform/double-checkbox-filterform.tsx

```diff
@@ -0,0 +1,160 @@
+                : endreFiltervalg(
+                      formCol1,
+                      checkBoxValgCol1.filter(value => value !== id)
+                  );
+        else if (typeForm === formCol2)
+            return e.target.checked
+                ? endreFiltervalg(formCol2, [...checkBoxValgCol2, id])
+                : endreFiltervalg(
+                      formCol2,
+                      checkBoxValgCol2.filter(value => value !== id)
+                  );
+        return;
+    };
+
+    const nullstillValg = () => {
+        endreFiltervalg(formCol1, []);
+        endreFiltervalg(formCol2, []);
+    };
+
+    return (
+        <form className="skjema checkbox-filterform">
+            {harValgCol1 && harValgCol2 && (
+                <div className={classNames('checkbox-filterform__valg__double', className)}>
+                    <div
+                        className="checkbox-filterform-col1"
+                        role="group"
+                        aria-labelledby="double-filterform-label-col1"
+                    >
+                        <Element id="double-filterform-label-col1" className="double-form-title">
+                            {'Er utdanningen godkjent i Norge?'}
+                        </Element>
+                        <RenderFields
+                            valg={uniqueValgCol1}
+                            form={formCol1}
+                            velgCheckBox={e => velgCheckBox(e, formCol1)}
+                            checkBoxValg={checkBoxValgCol1}
+                        />
+                    </div>
+                    <div
+                        className="checkbox-filterform-col2"
+                        role="group"
+                        aria-labelledby="double-filterform-label-col2"
+                    >
+                        <Element id="double-filterform-label-col2" className="double-form-title">
+                            {'Er utdanningen bestått?'}
+                        </Element>
+                        <RenderFields
+                            valg={uniqueValgCol2}
+                            form={formCol2}
+                            velgCheckBox={e => velgCheckBox(e, formCol2)}
+                            checkBoxValg={checkBoxValgCol2}
+                        />
+                    </div>
+                </div>
+            )}
+            <div className={'filterform__under-valg'}>
+                <NullstillValgKnapp
+                    dataTestId="double-checkbox-filterform"
+                    nullstillValg={nullstillValg}
+                    form={'utdanning-godkjent-og-bestatt'}
+                    disabled={!(checkBoxValgCol1.length > 0 || checkBoxValgCol2.length > 0)}
+                />
+                {(!harValgCol1 || !harValgCol2) && (
+                    <AlertStripe type="info" className="checkbox-filterform__alertstripe">
+                        {emptyCheckboxFilterFormMessage || 'Ingen veiledere funnet'}
```

Comment:
I am not sure that default value should be: 'ingen veiledere funnet', for this file it doesnt seems its related to veiledere..

Reply:
I'll look into it!